catalog/ui/catalog-ui-search/src/main/webapp/component/result-add/result-add.view.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define, require, module*/
+var Marionette = require('marionette');
+var $ = require('jquery');
+var template = require('./result-add.hbs');
+var CustomElements = require('js/CustomElements');
+var store = require('js/store');
+var ListCreateView = require('component/list-create/list-create.view');
+var lightboxInstance = require('component/lightbox/lightbox.view.instance');
+var List = require('js/model/list');
```

Comment:
should be js/model/List (capital L).... breaks on RHEL deployments otherwise

Reply:
Good catch.